get-started/rw-premium-edition-intro.mdx

```diff
@@ -48,6 +52,15 @@ To set your license key:
 ALTER SYSTEM SET license_key TO '...';
 ```
 
+#### Automated method
+
+To set your license key automatically:
+
+1. Use the `--license-key-file` CLI option for the meta node to monitor and reload the license key from a specified file. This streamlines license key rotation in Cloud environments.
```

Comment:
`--license-key-file` was later renamed to `--license-key-path` to be consistent with the env var key in https://github.com/risingwavelabs/risingwave/pull/18838. Would you help to update it? Same for other occurrences.